VDKQueue.m

```diff
@@ -422,6 +416,23 @@ - (void) removeAllPaths
 }
 
 
+- (void) stopWatching
+{
+    // This method must be called if we want this object to ever be dealloc'd. This is because detachNewThreadSelector:toTarget:withObject:
+    // retains the target (in this case, self), setting the retainCount to 2. Aside from the memory leak issue, if the thread is never terminated,
+    // the watcher thread might still try to send messages to a nonexistent delegate, causing your app to crash.
+    @synchronized(self)
+    {
+        // Shut down the thread that's scanning for kQueue events
+        _keepWatcherThreadRunning = NO;
+
+        // Do this to close all the open file descriptors for files we're watching
+        [_pathMap removeAllObjects];
```

Comment:
Maybe you were on a fork, because I don't think that `pathMap` exists.

Reply:
oops, yes, just disregard that line!